.github/workflows/ur-precommit.yml

```diff
@@ -99,6 +99,12 @@ jobs:
       docker_image: ${{ matrix.docker_image || 'ghcr.io/intel/llvm/ubuntu2404_intel_drivers:alldeps'}}
       image_options: ${{ matrix.image_options || '' }}
 
+  offload_build:
+    name: Adapters (Offload)
```

Comment:
nit: can we call it `Offload Adapter` or something?

Reply:
@kbenzie expressed a preference for this name here: https://github.com/intel/llvm/pull/18700#discussion_r2111996440

Reply:
To make it look consistent with the other adapter jobs ![image](https://github.com/user-attachments/assets/65c53a9e-05dc-4391-9de1-e2c42f10febc)

Reply:
ah ok got it